drivers/dma/Kconfig.stm32

```diff
@@ -4,6 +4,10 @@
 # Copyright (c) 2019 Song Qiang <songqiang1304521@gmail.com>
 # SPDX-License-Identifier: Apache-2.0
 
+DT_COMPAT_ST_STM32_DMA_V1 := $(dt_compat_enabled,$(st,stm32-dma-v1))
+DT_COMPAT_ST_STM32_DMA_V2 := $(dt_compat_enabled,$(st,stm32-dma-v2))
+DT_COMPAT_ST_STM32_DMAMUX := $(dt_compat_enabled,$(st,stm32-dmamux))
```

Comment:
The `$(st,stm32-dma-v1)` part doesn't look right to me. There's no kconfigfunction named `st`.

Are you sure about this?

Reply:
Late untested fix. Txs !